person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @parent_permission = parent_permission
+    @age = age
+  end
+
+  def can_use_services?
+    of_age? && @parent_permission == true
+  end
+
```

Comment:
- [ ] The `can_use_services` should return `true` if the method `of_age` returns `true` or `@parent_permission` is equal to `true`. Kindly correct the logic. 👍